prisma/migrations/20230809114415_add_missing_map/migration.sql

```diff
@@ -0,0 +1,27 @@
+/*
+  Warnings:
+
+  - You are about to drop the `Image` table. If the table is not empty, all the data it contains will be lost.
+
+*/
+-- DropForeignKey
+ALTER TABLE "Image" DROP CONSTRAINT "Image_recipe_id_fkey";
+
+-- DropTable
+DROP TABLE "Image";
+
+-- CreateTable
+CREATE TABLE "image" (
+    "id" SERIAL NOT NULL,
+    "name" TEXT NOT NULL,
+    "key" TEXT NOT NULL,
+    "recipe_id" INTEGER NOT NULL,
```

Comment:
this relation should not be `NOT NULL` there can be no recipe when you create image record in the database